torch/jit/__init__.py

```diff
@@ -1988,14 +1990,81 @@ def forward(self, input):
         """)
 
 
+def _unwrap_optional(x):
+    assert x is not None, "Unwrapping null optional"
+    return x
+
 _builtin_table = None
 
 _modules_containing_builtins = (torch, torch._C._nn)
 
-
-def _unwrap_optional(x):
-    assert x is not None, "Unwrapping null optional"
-    return x
+_builtin_ops = [
```

Comment:
why was this changed in the PR?

Reply:
to be accessible from `torch/jit/supported_ops.py`